### PR TITLE
add missing provisioning params to foreman_host

### PR DESCRIPTION
**Pull request by evgeni**
Fixes: #758

### CODE REVIEW

**Comment by evgeni** [comment]
@hlawatschek as I think you're using image based provisioning, mind taking a look here?

**Comment by evgeni** [comment]
after reading foreman source code, I don't think capabilities should be writeable and thus dropped it for now

**Comment by evgeni** [comment]
and I opened https://projects.theforeman.org/issues/29420 to fix that in foreman :)

File: plugins/modules/foreman_host.py [hunk 1]
```diff
@@ -98,6 +98,21 @@
       - Mutually excluside with I(owner).
     type: str
     required: false
+  provision_method:
```
Comment (mdellweg) [inline plugins/modules/foreman_host.py]: And this is not available on host_groups?

Comment (evgeni) [reply plugins/modules/foreman_host.py]: Correct

**Pull request merged**